var/spack/repos/builtin/packages/scorep/package.py

```diff
@@ -154,7 +154,7 @@ def find_libpath(self, libname, root):
         return libs.directories[0]
 
     def with_or_without(self, arg):
-        return super.with_or_without(arg).remove_suffix("=yes")
+        return super.with_or_without(arg).rstrip("=yes")
```

Comment:
```suggestion
        return super.with_or_without(arg).replace("=yes", "")
```
...rstrip: wrong semantics. remove_suffix: wrong. removesuffix: valid since 3.9. replace: always valid.